---
 kernel/sched/bfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-4.7-ck5/kernel/sched/bfs.c
===================================================================
--- linux-4.7-ck5.orig/kernel/sched/bfs.c	2016-09-23 08:33:00.215724491 +1000
+++ linux-4.7-ck5/kernel/sched/bfs.c	2016-09-23 08:33:00.212724509 +1000
@@ -137,7 +137,7 @@
 
 void print_scheduler_version(void)
 {
-	printk(KERN_INFO "BFS CPU scheduler v0.490 by Con Kolivas.\n");
+	printk(KERN_INFO "BFS CPU scheduler v0.497 by Con Kolivas.\n");
 }
 
 /*
